roles/reproducer/tasks/configure_controller.yml

```diff
@@ -379,9 +379,8 @@
           cifmw_architecture_automation_file: >-
             {{
               (
-                ansible_user_dir,
-                'src/github.com/openstack-k8s-operators',
-                'architecture/automation/vars',
+                cifmw_architecture_repo | default(ansible_user_dir+'/src/github.com/openstack-k8s-operators/architecture'),
```

Comment:
ansible_user_dir will reflect the value from the hypervisor - not a big deal in CI since we're using `zuul` everywhere, but in custom labs.... that will crash.

Reply:
so it is already broken for them? ansible_user_dir is already in the code. no?

Reply:
oh - dang, I missed the fact it was between {% raw %} - meaning it's all good. Sorry for that one!